src/amo/pages/UserProfileEdit/index.js

```diff
@@ -874,12 +877,12 @@ export function mapStateToProps(state: AppState, ownProps: Props) {
     isUpdating: state.users.isUpdating,
     lang,
     user,
-    username: user ? user.username : params.username,
+    userId: user ? user.id : userId,
```

Comment:
We could end up with a `userId` here that is `NaN`, which (I think) could cause problems in the component as I believe we're assuming that `userId` is a number in the component code, for example at line 124.

Reply:
This page is not public. The next iteration #6915 should fix that.